Makefile

```diff
@@ -15,3 +24,6 @@ test:
 
 clean:
 	rm -f $(TARGET)
+
+install:
+	install -D $(TARGET) $(LIBEXECDIR)/kata-containers/$(TARGET)
```

Comment:
`mkdir` first?

Reply:
`install -D` will actually create the destination directory.

I wonder if we want to standardise on always specifying owner, group and perms for all kata install rules?:

```
INSTALL_USER = root
INSTALL_GROUP = root
INSTALL_MODE = 0644

    :

install:
    install --owner="$(INSTALL_USER)" --group="$(INSTALL_GROUP)" --mode="$(INSTALL_MODE)" -D ...
```

Reply:
Good question. We can start with something simple first and open an issue to discuss more about that, WDYT @jodh-intel  ?